Library/Homebrew/cmd/update-report.rb

```diff
@@ -126,131 +127,137 @@ def output_update_report
+          hub.dump(updated_formula_report: !args.auto_update?) unless args.quiet?
+          hub.reporters.each(&:migrate_tap_migration)
+          hub.reporters.each { |r| r.migrate_formula_rename(force: args.force?, verbose: args.verbose?) }
 
-        CacheStoreDatabase.use(:descriptions) do |db|
-          DescriptionCacheStore.new(db)
-                               .update_from_report!(hub)
-        end
-        CacheStoreDatabase.use(:cask_descriptions) do |db|
-          CaskDescriptionCacheStore.new(db)
-                                   .update_from_report!(hub)
+          CacheStoreDatabase.use(:descriptions) do |db|
+            DescriptionCacheStore.new(db)
+                                 .update_from_report!(hub)
+          end
+          CacheStoreDatabase.use(:cask_descriptions) do |db|
+            CaskDescriptionCacheStore.new(db)
+                                     .update_from_report!(hub)
+          end
         end
+        puts if args.auto_update?
+      elsif !args.auto_update? && !ENV["HOMEBREW_UPDATE_FAILED"] && !ENV["HOMEBREW_MIGRATE_LINUXBREW_FORMULAE"]
+        puts "Already up-to-date." unless args.quiet?
       end
-      puts if args.auto_update?
-    elsif !args.auto_update? && !ENV["HOMEBREW_UPDATE_FAILED"] && !ENV["HOMEBREW_MIGRATE_LINUXBREW_FORMULAE"]
-      puts "Already up-to-date." unless args.quiet?
-    end
 
-    Commands.rebuild_commands_completion_list
-    link_completions_manpages_and_docs
-    Tap.each(&:link_completions_and_manpages)
+      Commands.rebuild_commands_completion_list
+      link_completions_manpages_and_docs
+      Tap.each(&:link_completions_and_manpages)
 
-    failed_fetch_dirs = ENV["HOMEBREW_MISSING_REMOTE_REF_DIRS"]&.split("\n")
-    if failed_fetch_dirs.present?
-      failed_fetch_taps = failed_fetch_dirs.map { |dir| Tap.from_path(dir) }
+      failed_fetch_dirs = ENV["HOMEBREW_MISSING_REMOTE_REF_DIRS"]&.split("\n")
+      if failed_fetch_dirs.present?
+        failed_fetch_taps = failed_fetch_dirs.map { |dir| Tap.from_path(dir) }
 
-      ofail <<~EOS
-        Some taps failed to update!
-        The following taps can not read their remote branches:
-          #{failed_fetch_taps.join("\n  ")}
-        This is happening because the remote branch was renamed or deleted.
-        Reset taps to point to the correct remote branches by running `brew tap --repair`
-      EOS
+        ofail <<~EOS
+          Some taps failed to update!
+          The following taps can not read their remote branches:
+            #{failed_fetch_taps.join("\n  ")}
+          This is happening because the remote branch was renamed or deleted.
+          Reset taps to point to the correct remote branches by running `brew tap --repair`
+        EOS
+      end
     end
 
     return if new_repository_version.blank?
 
     puts
     ohai "Homebrew was updated to version #{new_repository_version}"
```

Comment:
This also feels like something that `--quiet` should omit.

Reply:
I agree with the --quiet addition. I will add it.